src/styles/components/_about-us-introduction.scss

```diff
@@ -0,0 +1,95 @@
+.about-us-introduction {
+  display: flex;
+  flex-direction: column;
+  justify-content: space-between;
+  height: rem(250px);
+
+  @include respond-above('sm') {
+    height: rem(400px);
+  }
+  @include respond-above('md') {
+    height: rem(500px);
+  }
+
+  @include respond-above('lg') {
+    height: rem(600px);
+  }
+  @include respond-above('xl') {
+    height: rem(700px);
+  }
+
+  &__title {
+    width: 58%;
+    height: 12.5%;
+    margin-block-start: 0%;
+
+    color: $dark;
+    -webkit-text-stroke: rem(1px) $dark;
+
+    @include respond-above('md') {
+      font-size: rem(28px);
+    }
+
+    @include respond-above('lg') {
+      font-size: rem(32px);
+    }
+
+    @include respond-above('xl') {
+      width: 45%;
+
+      font-size: rem(36px);
+    }
+
+    font-size: rem(24px);
+    font-weight: bold;
+    font-stretch: normal;
+    font-style: normal;
+    line-height: 1.22;
+    letter-spacing: normal;
+    text-align: left;
+  }
+
+  &__description {
+    transform: translateY(-10%);
+    align-self: flex-end;
+    box-sizing: border-box;
+    width: 55%;
+    height: 55%;
+    padding: 3% 5% 2% 2.5%;
+
+    color: $light;
+    background-color: black;
+
+    @include respond-above('md') {
+      font-size: rem(16px);
+    }
+
+    @include respond-above('lg') {
+      font-size: rem(20px);
+    }
+
+    @include respond-above('xl') {
+      font-size: rem(24px);
+    }
+
+    font-size: rem(14px);
+    font-weight: 200;
+    font-stretch: normal;
```

Comment:
Same as this [comment](https://github.com/Cobda/cobda-web/pull/33/files#r583415810)